test/machine_install.py

```diff
@@ -83,10 +84,15 @@ def _serve_payload(self):
 
         return payload_cached_name, http_payload_port
 
-    def _write_interactive_defaults_ks(self):
+    def _write_interactive_defaults_ks(self, updates_image):
         payload_cached_name, http_payload_port = self._serve_payload()
         content = f'liveimg --url="http://10.0.2.2:{http_payload_port}/{payload_cached_name}"'
-        Machine.execute(self, f'echo \'{content}\' > /usr/share/anaconda/interactive-defaults.ks')
+        defaults_path = "usr/share/anaconda/"
+        with TemporaryDirectory() as tmp_dir:
+            os.makedirs(f"{tmp_dir}/{defaults_path}")
+            with open(f"{tmp_dir}/{defaults_path}/interactive-defaults.ks", "wt", encoding="utf-8") as f:
+                f.write(content)
+            os.system(f"cd {tmp_dir} && find . | cpio -c -o | gzip -9cv > {updates_image}")
```

Comment:
This does not extend the existing updates.img from what I understand - it overwrites it. Right?

Reply:
Oh, my true.... how could I do such a dumb mistake... I'll fix that, thanks for spotting this.

Reply:
Fixed!